manifests/w/Wondershare/PDFelement/Wondershare.PDFelement.yaml

```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: "Wondershare.PDFelement"
+PackageVersion: "8.0"
+PackageLocale: "en-US"
+Publisher: "Wondershare Software Co. Ltd."
+PackageName: "Wondershare PDFelement"
```

Comment:
```suggestion
PackageIdentifier: Wondershare.PDFelement
PackageVersion: 8.2.2
PackageLocale: en-US
Publisher: Wondershare
PackageName: Wondershare PDFelement
```
![image](https://user-images.githubusercontent.com/15158490/124385358-6c6a9080-dcd5-11eb-9681-87222611b8bd.png)

Reply:
from where this has been taken?
![](https://user-images.githubusercontent.com/15158490/124385358-6c6a9080-dcd5-11eb-9681-87222611b8bd.png)